### ISSUE TITLE
Animation skips to the end and goes into a loop

### ISSUE THREAD

**Issue by BestPresident** [ruffle-rs/ruffle#2043]
Could this be an AS3 thing?
[05fishanimal (2).zip](https://github.com/ruffle-rs/ruffle/files/5720083/05fishanimal.2.zip)

**Comment by Toad06** [comment]
Thanks for the report.

This is an AS2 file. The game crashes once it reaches `frame 111`, I think this is because of the presence of an unknown (non-ASCII?) character (displayed as `�` in JPEXS) in its `DoAction` code. 

Some messages are logged into the console:

`Error running definition tag: ExportAssets, got Invalid data: Invalid string data` _(displayed on startup)_

```
Couldn't parse SWF. This may or may not be a bug in Ruffle, please help us by reporting it to https://github.com/ruffle-rs/ruffle/issues and include the swf that triggered it.
No more actions will be executed in this movie.
```
_(displayed when frame 111 is reached)_

Compiling with #1499 only prevents the first error from being logged, so Ruffle still crashes.

Possibly the same issue as #1533.

**Comment by Herschel** [comment]
Yep, this is an SWFv5 text-decoding issue.

**Comment by Herschel** [comment]
Thanks for the report! Fixed by #2636.
